root/static/scripts/edit/components/edit/DiffSide.js

```diff
@@ -41,10 +48,23 @@ const DiffSide = ({
 }: Props): React.MixedElement | string => {
   const stack = [];
   const splitMatch = new RegExp('^(?:' + split + ')$');
-  const diffs = editDiff(
-    splitText(oldText, split),
-    splitText(newText, split),
-  );
+
+  let diffs;
+  if (
+    split === '' ||
```

Comment:
This line should actually be sufficient to fix the ticket, since AddAnnotation doesn't seem to use word diffing.  But perhaps we might want to change it to use word diffing, in which case the length check would be helpful.  (Note that splitting the text by \s+ instead of '' would also mean there are less items to compare, so it would be faster.)